README.md

```diff
@@ -24,6 +24,8 @@
     : namespace to populate razeedeploy resources into (Default 'razeedeploy')
 -s, --file-source=''
     : url that razeedeploy-job should source razeedeploy resource files from (Default 'https://github.com')
+-ft, --file-template=''
```

Comment:
```suggestion
--ft, --file-template=''
```

Reply:
also need to add this to the help menu in src/install.js `if (argv.h || argv.help) { ...`

Reply:
fixed